doc/python/renderers.md

```diff
@@ -171,18 +175,17 @@ fig.show(renderer="png")
 ```
 
 ###### `pdf`
-This renderer displays figures as static PDF files. This is especially useful for notebooks that will be exported to PDF files using the LaTeX export capabilities of nbconvert.
+This renderer displays figures as static `.pdf` files. This is especially useful for notebooks that will be exported to `.pdf` files using the `LaTeX` export capabilities of `nbconvert`.
```

Comment:
here the usage of backticks is questionable also... "static PDF files" is acceptable, no? It's not clear why "static `.pdf` files" is clearer or better :)